contracts/zero-ex/contracts/src/transformers/bridges/BridgeProtocols.sol

```diff
@@ -44,4 +44,5 @@ library BridgeProtocols {
     uint128 internal constant COFIX       = 14;
     uint128 internal constant NERVE       = 15;
     uint128 internal constant MAKERPSM    = 16;
+    uint128 internal constant UNISWAPV3   = 17;
```

Comment:
Potential collision with BalancerV2, not sure which will drop first.

https://github.com/0xProject/protocol/pull/206/files#diff-a3b6c2f58667ec3b4470e5cf373fb1b231938288b21d60fbdfbaa0730e6779d2R47